base/shell/explorer/lang/cs-CZ.rc

```diff
@@ -28,6 +28,19 @@ BEGIN
     END
 END
 
+IDM_TASKGRP MENUEX
```

Comment:
@StartForKiller Please change all your `IDM_TASKGRP MENUEX` to `IDM_TASKGRP MENU` in the resource files to fix the builds. Indeed, `MENUEX` resources have slightly different syntax for their `MENUITEM`s, which you don't make use here.